src/csharp/Extensions/Microsoft.Spark.Extensions.Hyperspace/Hyperspace.cs

```diff
@@ -20,6 +21,7 @@ public class Hyperspace : IJvmObjectReferenceProvider
         private readonly IJvmBridge _jvmBridge;
         private readonly JvmObjectReference _jvmObject;
 
+        [HyperspaceSince(HyperspaceVersions.V0_0_1)]
```

Comment:
nit: One naive question, if the version attribute has been added on the class level, does it need to be mentioned for every class API?

Reply:
Yeah. For example, a later version of Hyperspace could add some new functions to an existing class.